x-pack/plugins/rollup/public/crud_app/services/http_provider.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// This is an Angular service, which is why we use this provider pattern to access it within
+// our React app.
+let _http;
+
+export function setHttp(http) {
+  _http = http;
```

Comment:
Out of curiosity, why not use `kfetch`?

Reply:
I think we're blocked by https://github.com/elastic/kibana/issues/22594 (interceptor parity between kfetch and $http). We originally used kfetch and then switched to $http for this reason.